backend/app/src/main/java/com/moh/phlat/backend/service/dto/ReportSummary.java

```diff
@@ -0,0 +1,22 @@
+package com.moh.phlat.backend.service.dto;
+
+public class ReportSummary {
```

Comment:
The Name is too general. Please give some business name to this class. It is not indicating which report?

Reply:
I verified with our BA (Linda) and she wants to call the as a Report Summary.